Makefile

```
@@ -3,13 +3,13 @@
 PYMODULE:=april_vision
 TESTS:=tests
 
-all: lint type build
+all: type lint isort-check build
```

I prefer to run linting and isort first as these issues are generally readily solved
```suggestion
all: lint isort-check type build
```